<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLVolumeInfoWidget</class>
 <widget class="QWidget" name="qMRMLVolumeInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>366</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Volume Information</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ImageDimensionsLabel">
     <property name="text">
      <string>Image Dimensions:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkCoordinatesWidget" name="ImageDimensionsWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="coordinates">
      <string>1,1,1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ImageSpacingLabel">
     <property name="text">
      <string>Image Spacing:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkCoordinatesWidget" name="ImageSpacingWidget">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="coordinates">
      <string>1,1,1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ImageOriginLabel">
     <property name="text">
      <string>Image Origin:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ctkCoordinatesWidget" name="ImageOriginWidget">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="CenterVolumePushButton">
     <property name="text">
      <string>Center Volume</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ScanOrderLabel">
     <property name="text">
      <string>Scan Order:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="ScanOrderComboBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="NumberOfScalarsLabel">
     <property name="text">
      <string>Number of Scalars:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="NumberOfScalarsSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ScalarTypeLabel">
     <property name="text">
      <string>Scalar Type:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="ScalarTypeComboBox"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="ScalarRangeLabel">
     <property name="text">
      <string>Scalar Range:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="ScalarRangeLayout">
     <item>
      <widget class="QLabel" name="MinScalarLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Min:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="MinScalarDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MaxScalarLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Max:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="MaxScalarDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="FileNameLabel">
     <property name="text">
      <string>File Name:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="FileNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="LabelMapLabel">
     <property name="text">
      <string>LabelMap:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="LabelMapCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="WindowLevelPresetsLabel">
     <property name="text">
      <string>Window/Level Presets:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QListWidget" name="WindowLevelPresetsListWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCoordinatesWidget</class>
   <extends>QWidget</extends>
   <header>ctkCoordinatesWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
